### ISSUE TITLE
Angel like game play. [comment]

### ISSUE THREAD

**Issue by demethan** [davqvist/aoe#6]
I have hit the part with the different rocks and multi material. The mesh production is a bit of a problem.  The space required to make them is huge.  I would need to x5 that setup to just have enough mesh for my little rock setup. 

![image](https://github.com/user-attachments/assets/f57e79e1-a14f-4b88-aecb-6cbb504c8e8e)

![image](https://github.com/user-attachments/assets/084d21b9-fb07-4abb-905e-4ca802d4f9cc)

I am not a fan of Bob`s mod or Angel mod.  I never completed a playthrough with those mods in.  Not sure I am a good representation of the players you are trying to attract. 

Maybe I am not looking at this correctly.  I see that there will be some resource bees later and won`t have to rely on that primitive setup.

**Comment by demethan** [comment]
I see the flax flower is supposed to yield more fibre.  I am working towards that path now.

**Comment by davqvist** [comment]
Yes, the rock cycle is a bit inspired by angels, but I had a similar idea separately before even ever playing Angels. 
It is not quite the same and the automation needed to get your ores is totally different. But both require some form of logic with circuits and/or buffers to make it work that nothing backs up. I can understand you not liking it, but it is actually one of the things I like the most about my mod, so that part is unlikely to get changed. The rock cycle has a second part in age 4 where it gets more complicated. But yes in age 5 you could potentially make everything from bees if you so desire. But you have to use the rock cycle til age 5.

**Comment by demethan** [comment]
have any examples?

**Comment by davqvist** [comment]
Oh and also btw. You can do the rock cycle two different ways. You could turn all stone into lava and go from there if you wanna save on fibre meshes.

examples for what exactly?

**Comment by demethan** [comment]
of a rock setup.

**Comment by davqvist** [comment]
Sure if you don't mind my terrible spaghetti, even without any big buffering. Here stone into lava into the 3 stones on the left and then filtering the ores on the right. Ignore the other vertical belts.
![image](https://github.com/user-attachments/assets/4baa6a38-768b-4145-86a4-4c63b49bef67)

**Comment by davqvist** [comment]
Oh this is the second part though already. So just imagine the right strainers filtering into ore.
